kolga/libs/vault.py

```diff
@@ -34,9 +37,20 @@ def __init__(
                 message="VAULT_ADDR not defined. Skipping Vault usage.",
             )
 
-    def login(self, ci_jwt: str = settings.VAULT_JWT) -> None:
+    def login(self, ci_jwt: str = settings.VAULT_JWT, ci_jwt_private_key: str = settings.VAULT_JWT_PRIVATE_KEY) -> None:
         if self.initialized:
             try:
+                if ci_jwt_private_key:
+                    ci_jwt = encode(
```

Comment:
MyPy will not like this since `encode()` produces `bytes` and `ci_jwt` is set to `str`.

Reply:
It doesn't seem to do so:
type(jwt_token)
<class 'str'>

https://github.com/jpadilla/pyjwt/issues/391
https://github.com/jpadilla/pyjwt/commit/e3a1ba52b5926bd419a5bab73c64d7bce1089958
https://github.com/jpadilla/pyjwt/blob/master/jwt/api_jwt.py#L37-L44